_search-plugins/search-relevance/ml-inference-rerank-by-field.md

```diff
@@ -0,0 +1,277 @@
+{ "index": { "_id": 2 } }
+{ "borough": "Queens", "area_name": "Flushing", "description": "Flushing is a neighborhood in the northern part of Queens, famous for its Asian-American population and bustling business district.", "population": 227000, "facts": "Flushing is one of the most ethnically diverse neighborhoods in NYC, with a large Chinese and Korean population. It is also home to the USTA Billie Jean King National Tennis Center." } 
+{ "index": { "_id": 3 } }
+{ "borough": "Brooklyn", "area_name": "Williamsburg", "description": "Williamsburg is a trendy neighborhood in Brooklyn known for its hipster culture, vibrant art scene, and excellent restaurants.", "population": 150000, "facts": "Williamsburg is a hotspot for young professionals and artists. The neighborhood has seen rapid gentrification over the past two decades." } 
+{ "index": { "_id": 4 } }
+{ "borough": "Manhattan", "area_name": "Harlem", "description": "Harlem is a historic neighborhood in Upper Manhattan, known for its significant African-American cultural heritage.", "population": 116000, "facts": "Harlem was the birthplace of the Harlem Renaissance, a cultural movement that celebrated Black culture through art, music, and literature." } 
+{ "index": { "_id": 5 } }
+{ "borough": "The Bronx", "area_name": "Riverdale", "description": "Riverdale is a suburban-like neighborhood in the Bronx, known for its leafy streets and affluent residential areas.", "population": 48000, "facts": "Riverdale is one of the most affluent areas in the Bronx, with beautiful parks, historic homes, and excellent schools." } 
+{ "index": { "_id": 6 } }
+{ "borough": "Staten Island", "area_name": "St. George", "description": "St. George is the main commercial and cultural center of Staten Island, offering stunning views of Lower Manhattan.", "population": 15000, "facts": "St. George is home to the Staten Island Ferry terminal and is a gateway to Staten Island, offering stunning views of the Statue of Liberty and Ellis Island." }
+
+```
+{% include copy-curl.html %}
+
+## Step 2: Create a connector
+Create a conector assuming you have created a sagemaker model with a cross encoder 
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "SageMaker cross-encoder model",
+  "description": "Test connector for Sagemaker cross-encoder model",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "credential": {
+		"access_key": "<Access key>",
+		"secret_key": "<Secret key>",
+		"session_token": "<Session token>"
+  },
+  "parameters": {
+    "region": "us-east-1",
+    "service_name": "sagemaker"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "url": "<SageMaker Endpoint>",
+      "headers": {
+        "content-type": "application/json"
+      },
+      "request_body": "{ \"inputs\":${parameters.inputs}}" ,
+       "pre_process_function": 
```

Comment:
One pain point we solve in ML inference processor is to remove pre/post process function. Can you try if we can remove this pre_process_function ?

Reply:
Yep on removing it I was able to simplify the tutorial Thanks for the reccomendation